UPGRADE-15.0.md

```diff
@@ -288,6 +288,8 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 
 #### refactored different levels of error debugging on SF ([#3033](https://github.com/shopsys/shopsys/pull/3033))
 
+#### fix seo pages urls ([#3079](https://github.com/shopsys/shopsys/pull/3079))
+
```

Comment:
This should not be present in Storefront part of upgrade notes but framework ones

I would also mention something like this:
- this change adds migration `Version20240325165512.php` that changes friendly urls. If you do not want to change friendly urls in your project, add it as skipped to `migrations-lock.yaml`

Reply:
ups, I am blind, sorry. Also added description to upgrade log in https://github.com/shopsys/shopsys/pull/3079/commits/32c6914ef8e22c6a6466112b4b0ba665a9177838